pandas/tests/test_format.py

```diff
@@ -2449,6 +2449,17 @@ def test_to_csv_decimal(self):
         expected_float_format = ';col1;col2;col3\n0;1;a;10,10\n'
         self.assertEqual(df.to_csv(decimal=',',sep=';', float_format = '%.2f'), expected_float_format)
 
+    def test_to_csv_date_format(self):
+        # GH 10209
+        df = DataFrame({'A' : pd.date_range('20130101',periods=5,freq='s')})
```

Comment:
can you add a test with ``freq='D'`` as well (should be same results, so can do in a loop)